OLCNE/scripts/provision.sh

```diff
@@ -14,6 +14,7 @@
 
 # Constants
 readonly CERT_DIR=/etc/olcne/pki
+readonly EXTERNALIP_VALIDATION_CERT_DIR=/etc/olcne/pki/externalip-validation-webhook
```

Comment:
I don't think it is a great idea to have `EXTERNALIP_VALIDATION_CERT_DIR` as a sub-directory of the `CERT_DIR` directory (certificates at different levels in the hierarchy)